lnbits/extensions/gerty/crud.py

```diff
@@ -55,7 +55,10 @@ async def update_gerty(gerty_id: str, **kwargs) -> Gerty:
     await db.execute(
         f"UPDATE gerty.gertys SET {q} WHERE id = ?", (*kwargs.values(), gerty_id)
     )
-    return await get_gerty(gerty_id)
+
+    gerty = await get_gerty(gerty_id)
+    assert gerty
+    return gerty
```

Comment:
Nitpick @blackcoffeexbt, I think this function `update_gerty` should return types `Optional[Gerty]` in case the `Gerty` does not exist and the return values should be treated further down the line. Same as `get_gerty` does already.